assets/dev/js/editor/controls/media.js

```diff
@@ -60,6 +64,7 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 		this.setValue( {
 			url: '',
 			id: '',
+			fileName: '',
```

Comment:
We can't insert the filename inside the widget value. You can extract it from the file URL